module/VuFind/src/VuFind/Hierarchy/TreeRenderer/Factory.php

```diff
@@ -54,7 +54,8 @@ class Factory
     public static function getJSTree(ServiceManager $sm)
     {
         return new JSTree(
-            $sm->getServiceLocator()->get('ControllerPluginManager')->get('Url')
+            $sm->getServiceLocator()->get('ControllerPluginManager')->get('Url'),
+            $sm->getServiceLocator()->get('VuFind\Config')->get('config')
```

Comment:
We'll have to remember to update VuFind\Config to VuFind\Config\PluginManager here when merging the latest master.